src/main/java/lotto/domain/WinningLotto.java

```diff
@@ -17,15 +14,15 @@ public WinningLotto(List<Number> matchNumber, Number bonusNumber) {
     }
 
     public boolean isExistBonusNumber(List<Number> lottoNumbers) {
-        return Collections.frequency(lottoNumbers, bonusNumber) > 0 ? true : false;
+        return lottoNumbers.contains(bonusNumber);
     }
 
     public boolean isMatchNumber(Number lottoNumber) {
-        return Collections.frequency(matchNumber, lottoNumber) > 0 ? true : false;
+        return matchNumber.contains(lottoNumber);
     }
 
     public LottoResult getLottoMatchResult(List<LottoNumber> lottoList) {
-        Map<Rank, Integer> result = new HashMap<>();
+        EnumMap<Rank, Integer> result = new EnumMap<Rank, Integer>(Rank.class);
```

Comment:
사소하지만 구체적인 클래스보다는 추상적인 인터페이스에 의존해주시는 것이 좋습니다 ! :)

```suggestion
        Map<Rank, Integer> result = new EnumMap<>(Rank.class);
```